doc/features/gang-scheduling.md

```diff
@@ -11,20 +15,61 @@ spec:
        kubeScheduling: 
            scheduleTimeoutSeconds: 120
 ```
-Based on this configuration, RBG will automatically create a PodGroup CR; the PodGroup's minNumber equals the sum of all pods across all Roles in the RBG.
+Based on this configuration, RBG will automatically create a `PodGroup.scheduling.x-k8s.io` CR; the PodGroup's minNumber equals the sum of all pods across all Roles in the RBG.
+```yaml
+apiVersion: scheduling.x-k8s.io/v1alpha1
+kind: PodGroup
+metadata:
+  name: gang-scheduling
```

Comment:
change name from gang-scheduling to scheduler-plugin-gang, in the same format with Volcano